pkg/ddevapp/ddevapp_test.go

```diff
@@ -363,6 +363,21 @@ var (
 			DynamicURI:                    testcommon.URIWithExpect{URI: "/", Expect: "Super easy vegetarian pasta bake TEST PROJECT"},
 			FilesImageURI:                 "/sites/default/files/Logo.png",
 		},
+		// 20: Symfony
+		{
+			Name:                          "TestPkgSymfony",
+			SourceURL:                     "https://github.com/symfony/demo/archive/refs/tags/v2.6.0.tar.gz",
+			ArchiveInternalExtractionPath: "demo-2.6.0/",
+			Type:                          nodeps.AppTypeSymfony,
+			Docroot:                       "public",
+			Safe200URIWithExpectation:     testcommon.URIWithExpect{URI: "/robots.txt", Expect: "User-agent"},
+			DynamicURI:                    testcommon.URIWithExpect{URI: "/", Expect: "Symfony Demo"},
+			FilesImageURI:                 "/apple-touch-icon.png",
+			PretestCmd:                    "ddev composer install --no-progress",
```

Comment:
We usually fork the upstream project, and prepare everything inside, including the commit for the Composer vendor dependencies (which makes testing faster).

Example with Laravel https://github.com/ddev/test-laravel

I suggest you to fork https://github.com/symfony/demo/tree/v2.6.0 and commit vendor dependencies, then you can transfer your fork to DDEV namespace later.

Reply:
Or I can create a fork, if you like.

Reply:
i have forked the demo repo here https://github.com/IndraGunawan/test-symfony and commit the vendor dir
I can't transfer to ddev org
`You don’t have the permission to create public repositories on ddev`

Reply:
Thanks, I invited you to the `ddevl` org, you can transfer your repo there, and then I will transfer it to the `ddev` org.

(This is how we usually do this.)

Reply:
transfered

Reply:
Thanks, I did my part too. Let's update the URL in tests, and I'll restart them.